test/test_subscriber.cpp

```diff
@@ -63,7 +66,7 @@ TEST(Subscriber, simple)
 {
   auto node = std::make_shared<rclcpp::Node>("test_node");
   Helper h;
-  Subscriber<Msg> sub(node, "test_topic");
+  Subscriber<Msg> sub(node, "test_topic", g_qos_profile, g_options);
```

Comment:
Are these changes to the tests necessary? I figured that it shouldn't be given the default parameters.

Reply:
I think you're right: they're not necessary given default parameters.